src/test/test-mkdir.c

```diff
@@ -22,33 +22,33 @@ TEST(mkdir_p_safe) {
 
         ASSERT_NOT_NULL(p = path_join(tmp, "run/aaa/bbb"));
         ASSERT_OK(mkdir_p(p, 0755));
-        ASSERT_GT(is_dir(p, false), 0);
-        ASSERT_GT(is_dir(p, true), 0);
+        ASSERT_OK_POSITIVE(is_dir(p, false));
+        ASSERT_OK_POSITIVE(is_dir(p, true));
 
         p = mfree(p);
         ASSERT_NOT_NULL(p = path_join(tmp, "run/ccc/ddd"));
-        ASSERT_OK(mkdir_p_safe(tmp, p, 0755, UID_INVALID, GID_INVALID, 0));
-        ASSERT_GT(is_dir(p, false), 0);
-        ASSERT_GT(is_dir(p, true), 0);
+        ASSERT_OK_ERRNO(mkdir_p_safe(tmp, p, 0755, UID_INVALID, GID_INVALID, 0));
```

Comment:
`mkdir_p_safe()` is provided by us, and return an negative errno. For such functions, please use `ASSERT_OK()`.

But, `symlink()` is provided by glibc, which set errno on failure. For such functions, please use `ASSERT_OK_ERRNO()`.

So, in this case, please use `ASSERT_OK()`, rather than `ASSERT_OK_ERRNO()`